libafl/src/fuzzer/mod.rs

```diff
@@ -833,22 +819,22 @@ where
     ) -> Result<ExitKind, Error>;
 }
 
-impl<CS, E, EM, F, OF> ExecutesInput<E, EM> for StdFuzzer<CS, F, OF, E::Observers>
+impl<CS, E, EM, F, OF, OT> ExecutesInput<E, EM> for StdFuzzer<CS, F, OF, OT>
```

Comment:
Why add new generics?

Reply:
because this OT doesn't have to be E's observers

Reply:
this OT is not a a new generic,
StdFuzzer is just defined as StdFuzzer<CS, F, OF, OT>. and there's no reason that this one should have E::Observers. and other part of this file have OT. right?

If we want to say OT is always E::Observers, then it's just better to remove OT at all. and define UsesObservers.
This is the exactly what I see as the problem. because there's no reason that this place should have E::Observers while the impl just above it has OT instead of it.